T100TA_guide.md

```diff
@@ -297,6 +299,8 @@ With hardware video decoding, a video player should use around 25% CPU when play
 ### 14. Bluetooth
 /!\ Same warning as Sound and WiFi, the following file is for T100TA and T100CHI only. Other T100's (T100TAF and T100H\*) have other Bluetooth device numbers.
 
+/!\ On T100CHI, to get keyboard working, boot with docking off and after boot you can switch on, wait it stop blinking and redo a swipe to right with this switch for pairing
```

Comment:
I'd rather move it in a troubleshooting section line 345
```
### 4. T100CHI Bluetooth keyboard
To get keyboard working, boot with docking off and after boot you can switch on, wait it stop blinking and redo a swipe to right with this switch for pairing.
```

By the way, I don't really understand the instructions, could you reformulate?
Au pire, écris-le en français et je le traduirai.

"wait it stop blinking" -> wait what?
"redo a swipe to right with this switch for pairing" -> ?

Disclaimer: I don't have the T100CHI.

Reply:
Yes, new troubleshooting section is a better place :+1: 
Pour illustrer le fonctionnement du clavier Bluetooth de la T100CHI, tu peux regarder la page 29 de ce lien : https://dlcdnets.asus.com/pub/ASUS/nb/T100CHI/040C_F10150_T100CHI_V2_A.pdf
Il comporte un interrupteur ON/OFF, ainsi qu'une troisième position en poussant vers la droite pour appairer, une fois relâché l'interrupteur revient sur ON.
Sur cette page 29, les étapes 1. et 2. doivent marcher Windows mais pas sous Linux.
J'avais déjà repéré ce comportement sur Debian et Ubuntu 16.04, ce clavier est plus capricieux que d'autres appareils Bluetooth.
Il faut démarrer la tablette avec le dock sur OFF, aller dans les paramètres Bluetooth (sur Gnome pour mon installation), allumer le dock (ON), attendre qu'il arrête de clignoter puis pousser cet interrupteur vers la droite environ deux secondes pour le voir apparaître dans la liste des périphériques Bluetooth détectés. Ensuite les étapes 3. et 4. peuvent être suivies.
Sans ces manipulations dans cet ordre, impossible de se connecter et au début il y a de quoi s'arracher les cheveux, car si on se loupe, la seule solution est de redémarrer la tablette et le dock.
En espérant avoir été plus clair !